### ISSUE TITLE
Setup page log warnings

### ISSUE THREAD

**Issue by PVince81** [owncloud/core#16685]
### Steps
1. Open the setup page

### Expected
No warning

### Actual
```
{"reqId":"YHm2oXUePZQoh3b1zqNU","remoteAddr":"172.17.42.1","app":"PHP","message":"Undefined offset: 0 at \/srv\/www\/htdocs\/owncloud\/lib\/private\/appframework\/http\/request.php#683","level":3,"time":"2015-06-02T12:15:22+00:00","method":"GET","url":"\/owncloud\/index.php\/apps\/files\/"}
```

### Versions
ownCloud 8.1 git master (3ebc8f0564395fbb519fc4275e0f6f883df708b7)

@LukasReschke any idea ?

**Comment by PVince81** [comment]
Bisecting now... looks like it's very old

**Comment by PVince81** [comment]
Had a chat with @LukasReschke who might have an idea. Assigning to him.

**Comment by RealRancor** [comment]
Looks to me like a dup of https://github.com/owncloud/core/issues/15992 ?

**Comment by PVince81** [comment]
Let's keep it separate.

This ticket here is about the offset 0.

And the other can be kept for the "touch()" part.

**Comment by RealRancor** [comment]
The touch() part is already handled in https://github.com/owncloud/core/issues/12893. So https://github.com/owncloud/core/issues/15992 is mixing both.

**Comment by PVince81** [comment]
I closed https://github.com/owncloud/core/issues/15992

**Comment by RealRancor** [comment]
Works :) The main point about the possible reasons for this was posted by @Raydiation in https://github.com/owncloud/core/issues/15992#issuecomment-97966255:

> Error happens here https://github.com/owncloud/core/blob/master/lib/private/appframework/http/request.php#L683
>
> Apparently is caused by having no trusted_domains setting in config/config.php which could be because initially there are no trusted domains

**Comment by PVince81** [comment]
Yes, @LukasReschke told me the same thing, and that he would have a look :smile:

**Comment by MorrisJobke** [comment]
:+1:

**Comment by davitol** [comment]
:+1: Fixed with PR #17078
